packages/as-types/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to as-types
+Thank you for considering contributing to as-types !
+
+## Reporting Bugs
+If you discover a bug, please create a [new issue](https://github.com/massalabs/as/issues/new?assignees=&labels=issue%3Abug&template=bug.md&title=) on our GitHub repository.
+In your issue, please include a clear and concise description of the bug, any relevant code snippets, error messages, and steps to reproduce the issue.
+
+## Installation
+To start developing with as-types, you must install all the necessary dev dependencies. You can do so by running the following command:
+
+```sh
+npm install
+```
+
+This will install all the required packages listed in the package.json file, allowing you to update, fix, or improve as-types in any way you see fit. 
+
+## Contributing Code
+We welcome contributions in the form of bug fixes, enhancements, and new features.
+
+To contribute code, please follow these steps:
+
+1. Fork the As repository to your own account.
+2. Create a new branch from the `main` branch for your changes.
+3. Make your changes and commit them to your branch.
+4. Push your branch to your fork.
+5. Create a pull request from your branch to the develop branch of the As repository.
+
+> **NOTE:** When creating a pull request, please include a clear and concise title and description of your changes, as well as any relevant context or background information.
+
+## Code Style
+Please ensure that your code follows the existing code style used in the project.
+We use the [MassaLabs Prettier configuration](https://github.com/massalabs/prettier-config-as) and [MassaLabs ESLint configuration](https://github.com/massalabs/eslint-config) for formatting and linting.
+
+You can run the following command to format your code before committing:
+
+```sh
+npm run fmt
+```
+
+## Tests
```

Comment:
We should tlak bout the fact this this is a mono repo and that the test and the CI should be started from the parent directory.
Maybe there is also other consideration to keep in mind that I'm not aware of